homeassistant/components/homeassistant_analytics_consumer/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "homeassistant_analytics_consumer",
+  "name": "Home Assistant analytics sensor",
+  "codeowners": ["@joostlek"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/homeassistant_analytics_consumer",
+  "integration_type": "service",
+  "iot_class": "cloud_polling",
+  "loggers": ["python-homeassistant-analytics"],
+  "requirements": ["python-homeassistant-analytics==0.3.0"]
```

Comment:
The library should check the e-tag and save bandwidth if the data is not updated

Reply:
Can we save that for a followup?

Reply:
Please implement it with this PR